htdocs/public/payment/paymentok.php

```diff
@@ -632,7 +635,7 @@
 
 				if (! $error)
 				{
-					$paiement_id = $paiement->create($user, 1);    // This include closing invoices and regenerating documents
+					$paiement_id = $paiement->create($userpayment, 1);    // This include closing invoices and regenerating documents
```

Comment:
Having a track of a user to make an action when this user has done nothing is hurting me.
When we pay online, data are saved by none (anonymous outside a login context), so i would prefer to have the value not defined in database.
What is table/field that is blocking ?